src/utils/context.tsx

```diff
@@ -51,6 +55,53 @@ const ContextProvider = ({ children }: React.PropsWithChildren<{}>) => {
         return "❓"
     }
   }
+  const displayHabit = (habit: number) => {
+    switch (habit) {
+      case 1:
+        return "Exercise"
```

Comment:
Could this be a dictionary object or something similar? Like a key - value pair instead of a switch case?

Reply:
Modified function to an object.  A lot of extra work dealing with typescript yelling. Thank you Tanner